### PR TITLE
Fix share button on web preview

### LINKED ISSUE #442: "Share" button does not work on web preview
I get the following error:

```
MissingPluginException(No implementation found for method share on channel plugins.flutter.io/share)
```

### PR DESCRIPTION
**Pull request by epicfaace**
Making a PR based on @pieteraelse's branch

## What does this PR accomplish?
Fixes #442 by fixing the share button on web preview. We may need to fix this given that we need a public web app for master.

## Did you add any dependencies?

No

## How did you test the change?

Web preview

### CODE REVIEW

File: client/flutter/lib/home_page.dart [hunk 1]
```diff
@@ -2,101 +2,134 @@ import 'package:WHOFlutter/generated/l10n.dart';
...
 import 'package:flutter_staggered_grid_view/flutter_staggered_grid_view.dart';
 import 'package:share/share.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import 'constants.dart';
 
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Material(
```
Comment (creativecreatorormaybenot) [inline client/flutter/lib/home_page.dart]: This `Material` makes no sense (`Scaffold` adds a `Material` already).

File: client/flutter/lib/home_page.dart [hunk 2]
```diff
@@ -100,6 +105,29 @@ class HomePage extends StatelessWidget {
       ),
     );
   }
+
+  Future<void> _tryShareOrShowSnackBar(BuildContext context) async {
+    const url = 'https://www.who.int/covid-19-app';
+    const shareText = 'Check out the official COVID-19 Guide App $url';
+
+    try {
+      await Share.share(shareText, subject: 'Official COVID-19 Guide App');
```
Comment (epicfaace) [inline client/flutter/lib/home_page.dart]: Can we remove the subject for now:

```suggestion
      await Share.share(shareText);
```

**Comment by PieterAelse** [comment]
@epicfaace Indeed, I can comment again and PR again. So I'll have a look and open a PR on my own after that 👍 . I'll take the suggested changes along, thanks for opening the PR and checking it!

**Comment by PieterAelse** [comment]
@epicfaace @SamMousa 
I've had a look into the first-click issue. For some reason Share isn't throwing an exception the first time, also reproducible on `master`. See this screenshot for example:
Screen Shot 2020-03-28 at 21 06 31


So I've went for another approach, PR is open: https://github.com/WorldHealthOrganization/app/pull/507

**Comment by epicfaace** [comment]
Great, closing this PR then.

**Pull request closed without merging**